third_party/ibis/ibis_postgres/client.py

```diff
@@ -13,11 +13,20 @@
 # limitations under the License.
 
 # from ibis.backends.postgres.client import PostgreSQLClient
+
+from __future__ import annotations
+import parsy
+import re
+import ast
+import toolz
 import ibis.expr.datatypes as dt
 import ibis.expr.schema as sch
+
+# import ibis.expr.types as ir
```

Comment:
Is this to be removed?

Reply:
Yes, I will clean this up. I added it during development but ir wasn't needed in the final edit.